domain-models-runtime/src/main/java/org/folio/rest/persist/PostgresClient.java

```diff
@@ -2705,6 +2758,22 @@ public void select(String sql, Handler<AsyncResult<RowSet<Row>>> replyHandler) {
     getSQLConnection(conn -> select(conn, sql, closeAndHandleResult(conn, replyHandler)));
   }
 
+    /**
+   * Run a select query.
+   *
+   * <p>To update see {@link #execute(String, Handler)}.
+   *
+   * @param sql - the sql query to run
+   * @param replyHandler the query result or the failure
+   * @param queryTimeout query timeout in milliseconds, or 0 for no timeout
+   */
+    public void select(String sql, Handler<AsyncResult<RowSet<Row>>> replyHandler,
+        int queryTimeout) {
```

Comment:
Can you move the replyHandler parameter to the end? That will be more consistent with all other methods and also much more readable when called because the replyHandler may be a multi-line argument. It will also ensure a smooth upgrade path to the Vert.x 4 futurisation.

Reply:
done